rest-api-spec/src/main/resources/rest-api-spec/api/_internal.prevalidate_node_removal.json

```diff
@@ -0,0 +1,35 @@
+{
+  "_internal.prevalidate_node_removal": {
+    "documentation":{
+      "url": "https://www.elastic.co/guide/en/elasticsearch/reference/master/prevalidate-node-removal-api.html",
+      "description":"Prevalidates node removal from the cluster"
+    },
+    "stability":"experimental",
```

Comment:
I think we should call this stable before release.

Reply:
Why? What would be the requirement to call it stable? I thought we start with experimental and at some point call it stable. Do we have some guidelines about this?

Reply:
Most features start as stable. In this case, the primary consumer is orchestration and as such we need to be confident enough in the feature to release it GA, i.e., fully support it and only make bwc changes to it in the future.

We can also hide this behind a feature flag until it is ready but I see little value in getting it out as experimental since we will be very unlikely to receive feedback affecting the design.

Reply:
Thanks for the explanation. I've changed to stable. Frankly, I imagined we'd leave this experimental until at lease the second part of this API is merged, especially since this is only for internal uses I thought that would be possible w/o hindering adoption. Nonetheless, for now I cannot think of any non-BWC changes that the second part could introduce. It probably adds a timeout parameter and some of those NOs get refined into YES/NO/UNKNOWNs.